tests/test_run_state_async.py

```
@@ -0,0 +1,90 @@
+    assert len(runs_async) == 5
+    assert {r["run_id"] for r in runs_async} == set(ids)
+
+
+def test_concurrent_access(tmp_path):
+    async def _test():
+        manager = RunStateManager(runs_root=tmp_path)
+        state = await manager.create_run(flow_id="concurrent-flow")
+        run_id = state["run_id"]
+
+        # Simulate concurrent updates
+        async def update_status(status):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `update_status`

Add return type annotation: `None`

(ANN202)